.docs/content/2.usage/use-armonik-api-in-an-angular-app.md

```diff
@@ -0,0 +1,549 @@
+---
+description: This page will guide you through the process of using ArmoniK API in an Angular App.
+---
+
+# Use ArmoniK API in an Angular App
+
+The purpose of this guide is to show you how to use ArmoniK API in an Angular App. We will guide you through the process of creating a new Angular App and install ArmoniK API in it and use it.
```

Comment:
The purpose of this guide is to explain how to use ArmoniK API in an Angular App. We will guide you through the process of creating a new Angular App and install ArmoniK API in it and use it.